packages/pharos/src/components/image-card/pharos-image-card.ts

```diff
@@ -364,12 +364,13 @@ export class PharosImageCard extends ScopedRegistryMixin(FocusMixin(PharosElemen
   private _cardToggleSelect(event: Event): void {
     if (
       !this.disabled &&
-      (this._isSelectableViaCard() || (event.target as Element)?.nodeName == 'PHAROS-CHECKBOX')
+      ((this._isSelectableViaCard() && (event.target as Element)?.nodeName != 'PHAROS-CHECKBOX') ||
+        (event.currentTarget as Element)?.nodeName == 'PHAROS-CHECKBOX')
```

Comment:
This is of the form `(A ^ !B) v B)` which is equivalent to `A v B`:

```suggestion
      (this._isSelectableViaCard() ||
        (event.currentTarget as Element)?.nodeName == 'PHAROS-CHECKBOX')
```

Reply:
I feel like there might be something a little extra going on because of the event bubbling up. When the checkbox is clicked and handles the event here, which enables the card selection and `_isSelectableViaCard()` becomes `true`. 

Then as the event bubbles, the card handles the event and since `_isSelectableViaCard` is `true` it gets handled again and we deselect the card.